apps/hash-graph/bin/cli/src/subcommands/server.rs

```diff
@@ -260,19 +267,42 @@ pub async fn server(args: ServerArgs) -> Result<(), GraphError> {
 
     stop_gap_setup(&pool).await?;
 
-    let rest_router = rest_api_router(
-        Arc::new(pool),
-        DomainValidator::new(args.allowed_url_domain),
-    );
+    #[cfg(feature = "type-fetcher")]
+    let type_fetcher = {
+        let type_fetcher_address = format!("{}:{}", args.type_fetcher_host, args.type_fetcher_port);
+
+        let type_fetcher_address: SocketAddr = type_fetcher_address
+            .parse()
+            .into_report()
+            .change_context(GraphError)
+            .attach_printable_lazy(|| type_fetcher_address.clone())?;
+
+        let transport =
+            tarpc::serde_transport::tcp::connect(&type_fetcher_address, MessagePack::default);
+
+        FetcherClient::new(
+            client::Config::default(),
```

Comment:
We may as we can configure `max_in_flight_requests` and `pending_request_buffer`, so I suppose it would be good for tweaking. Likewise for the server config, but I'm not sure we need it right now

Reply:
No, probably we don't need it right now, but it's good to know that we can tweak the client a little bit without much work :)